src/main/java/com/csbank/v1/BankAccount.java

```diff
@@ -1,6 +1,46 @@
 package com.csbank.v1;
 
 public class BankAccount {
+    private int balance;
+
+    public BankAccount(int balance) {
+        this.balance = balance;
+    }
+
+    public int getBalance() {
+        return balance;
+    }
+
+    public boolean deposit(int deposit) {
+        if (deposit > 0){
+            balance = balance + deposit;
+        } else {
+            System.out.println("Invalid input.");
+            return false;
+        }
+        return true;
+    }
+
+    public boolean withdraw(int withdraw) {
+        if (withdraw <= 0) {
+            System.out.println("Invalid input.");
+            return false;
+        } else if (balance - withdraw < 0) {
+            System.out.println("Insufficient funds.");
+            return false;
+        } else {
+            balance -= withdraw;
+            return true;
+        }
+    }
+
+}
     // run the test, see the compilation error, and construct the class
     // based on the requirements stated in the test.
```

Comment:
you don't need this anymore. Please remove.  We should always keep our code "clean"